### ISSUE TITLE
Date range slider modal version and viewport-specific updates

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/dlme#770]
Depends on #764, mostly.

There are a couple of adjustments to the date range slider that ideally we would apply in specific contexts:

- In the "View larger" modal version, omit the "Unknown" value and associated count, if it exists. (Rationale is the user would have already see that there are some unknown values in the sidebar version before choosing the see the date slider in the modal version, and so the Unknown value count is probably not adding any new value in the modal version.)

- If we can do the above, we probably need to add a bottom margin to the input group to ensure there is some vertical spacing between the input and the bottom edge of the modal. See the screenshot below the next item for the ideal (I used `bottom-margin: 1rem;` on the input group's parent div).

- In both the  "View larger" modal version and in the sidebar version at `md` and `sm` viewport widths (which have a fairly wide content area available for the date slider), ensure that the new version of the input group + Apply button (to be implemented in #764) does not extend full-width of the content area, which would make the inputs too wide. There might not be any work required if the implementation of #764 works out such that the input group doesn't extend full-width (I'm not sure what it will do).

In other words, ensure that for the  "View larger" modal version and in the sidebar version at `md` and `sm` viewports, the inputs don't get much wider than this:

![Screen Shot 2019-12-03 at 4 49 27 PM](https://user-images.githubusercontent.com/101482/70102536-422e8980-15ed-11ea-8c75-5045f544cfd4.png)
